lib/classes/PluginManager.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const Utils = require('./Utils');
+const path = require('path');
+const has = require('lodash').has;
+const forEach = require('lodash').forEach;
```

Comment:
Good question! I asked @nikgraf the exact same question because I thought the same. But the lodash `forEach` can be run on objects as well (which is used by the new plugin management).